keyboards/espectro/readme.md

```diff
@@ -0,0 +1,16 @@
+ALU84
```

Comment:
```suggestion
# ALU84
```

Reply:
ALU84?

```suggestion
# Espectro
```

Reply:
@noroadsleft yup built this based off of my ALU84 build, missed that line.